docker-compose.yml

```diff
@@ -1,22 +1,24 @@
 web:
```

Comment:
We don't. It might be useful to ease the project installation for contributors, but then we'd have to test it somehow. I'd prefer to keep it for now, but if we don't figure out an easy way for testing, remove it later :+1: